py-rse/configuration.Rmd

```diff
@@ -425,26 +397,56 @@ if __name__ == '__main__':
                         help='Output image file name')
     parser.add_argument('--xlim', type=float, nargs=2, metavar=('XMIN', 'XMAX'),
                         default=None, help='X-axis limits')
-    parser.add_argument('--rcparams', type=str, default=None,
-                        help='YAML file containing new matplotlib runtime configuration parameters')
+    parser.add_argument('--plotparams', type=str, default=None,
+                        help='YAML file containing matplotlib parameters')
     args = parser.parse_args()
     main(args)
 ```
 
 ## Summary {#configuration-summary}
 
-Tweaking the default matplotlib settings is a great example of the different
-configuration options you can use in your programming.
-Adding command line options to your scripts is a natural starting point
-(and will be all the complexity you'll need in many cases),
-but system-wide and user-specific configuration can be a great option for standardising
-settings across your all or selected aspects of your personal work,
-while job-specific configuration can be a good option if you're looking to scale up
-to multiple users and/or a large number of configuration options. 
+Programs are only useful if they can be controlled,
+and work is only reproducible if those controls are explicit and shareable.
+If the number of controls needed is small,
+adding command-line options to programs and setting those options in Makefiles
+is usually the best solution.
+As the number of options grows,
+so too does the value of putting options in files of their own.
+And if we need to manage differences between different installations of the software,
+user-specific and system-wide configuration files are probably the best choice.
```

Comment:
I'm a little unclear on what this means? (i.e. I read this sentence and I'm still unclear about what is a good use case for user-specific and system-wide configurations)

Reply:
Changed the wording to emphasize that this is for sys admins to hide differences on multi-user setups (e.g. research clusters).